it/test/utils/IntegrationTest.scala

```diff
@@ -124,6 +124,11 @@ trait IntegrationTest extends AnyWordSpec with Matchers with GuiceOneServerPerSu
     "feature-switch.offPayrollWorking" -> "false"
   )
 
+  def configContentTailoringOn: Map[String, String] = config() ++ Map(
+    "feature-switch.tailoringEnabled" -> "true"
```

Comment:
This is not the right FS, also this is used by `appWithTailoringOn` but thats not used anywhere, so if this is not needed it can be removed as well.